### DISCUSSION TITLE
PlanetScale is not supporting my creditCard.

### DISCUSSION THREAD

**Discussion by Abhishek7Tech** [planetscale/discussion#564] [General]
Hi, I am from India. I am trying to use the free tier which requires a credit/debit card to create a database. 
When I entered my card details it wasn't working. 

I can use any other database but I am working on an open source project. So I have no other choice.

Please help.

**Comment by mscoutermarsh** [comment]
Hi @Abhishek7Tech,

Could you write into support? https://support.planetscale.com/hc/en-us. We'll be able to help you there. We can't discuss payment details on GitHub.

**Comment by Abhishek7Tech** [comment]
Hello @mscoutermarsh ,

Done ✅.
